api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
I don't really like the semantics of this, but I'm guessing it was a decision made a while ago-- just feels me that the exceptional case (unregistered contributor) appears as the `serializer_class`. '

The existence of `UnregisteredContributorSerializer` doesn't make a ton of sense to me. It's just a user + one extra field. But I'm probably just being overly zealous. 

Not a big deal, but this definitely threw me at first.